server/chat-commands/core.ts

```diff
@@ -1486,8 +1521,8 @@ exports.commands = {
 		} else {
 			let altCommandHelp;
 			let helpCmd;
-			let targets = target.split(' ');
-			let allCommands = Chat.commands;
+			const targets = target.split(' ');
+			const allCommands: AnyObject = Chat.commands;
```

Comment:
```suggestion
			const allCommands = Chat.commands;
```

This does not need a cast, just fix it in the implementation.

Reply:
Feel free to push the errors, I'll know how to fix them once GitHub Actions tells me what they are.